src/main/java/com/kau/capstone/global/common/Gender.java

```diff
@@ -0,0 +1,21 @@
+package com.kau.capstone.global.common;
+
+import jdk.jfr.ContentType;
+
+public enum Gender {
+    MALE(1), FEMALE(2);
+    private final Integer value;
+
+    Gender(int value) {this.value = value;}
+    public int getValue() {
+        return value;
+    }
+
+    // 정수를 ENUM으로 변환하는 메서드
+    public static Gender fromInt(int value) {
+        if(value == 1) return MALE;
+        else if(value == 2) return FEMALE;
+        else
+            throw new IllegalArgumentException("유효하지 않은 성별입니다.");
```

Comment:
P1: 여기는 어차피 `if` - `else if`문을 통과하면 에러를 반환하기 때문에 굳이 `else`를 명시적으로 할 필요는 없어보입니다.
`PetType`도 마찬가지입니다.

```suggestion
        if(value == 1) return MALE;
        else if(value == 2) return FEMALE;

        throw new IllegalArgumentException("유효하지 않은 성별입니다.");
```

Reply:
제 개인적인 의견이지만, 저는 if, else if, else를 모두 명시하는게 코드 가독성이 좋다고 생각합니다.. 그래서 else를 넣어두었는데, 혹시 어떻게 생각하시나요?

Reply:
저는 사실 프로젝트할 때 [객체지향 생활체조 원칙](https://catsbi.oopy.io/bf003ff6-2912-4714-8ac2-44eeb7becc93)을 최대한 적용해와서 else는 진짜 어쩔 수 없는 경우에만 쓴 기억 밖에 없어가지구요
코드 스타일이랑 많이 다르시면 else문은 그대로 두시면 될 것 같습니다

Reply:
아하,, 그렇군요. 이해했습니다
추천해주신대로 else 예약어 제외하여 코드 수정해두었습니다!